core/lib/dal/migrations/20240320173835_consistency-checker-cursor.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE consistency_checker_info
+(
+    last_processed_l1_batch BIGINT NOT NULL,
+    created_at              TIMESTAMP NOT NULL,
+    updated_at              TIMESTAMP NOT NULL
+);
+
+INSERT INTO consistency_checker_info(last_processed_l1_batch, created_at, updated_at)
+VALUES (1, NOW(), NOW());
```

Comment:
Shouldn't the processed L1 batch number be zero rather than 1? At least it makes sense from the semantics perspective – one can say that the 0-th / genesis L1 batch is always checked, but this is not necessarily the case for the 1st batch. Also, personally, IMO making the `last_processed_l1_batch` column nullable would better reflect the domain (it'd allow not to insert an artificial value into the table).

Reply:
Nice catch! I was changing semantics of this table and missed this place